ui/components/betting/Layout.tsx

```diff
@@ -0,0 +1,212 @@
+  setSelectedAmount,
+  setSelectedOutcomeToken,
+  selectedOutcomeToken,
+  selectedAmount,
+  buy,
+  sell,
+  redeem,
+}) => (
+  <>
+    <div>
+      <FormInput
+        variant="filled"
+        label="Collateral value (WETH)"
+        type="number"
+        onChange={(e) => setSelectedAmount(e.target.value)}
+        disabled={isMarketClosed}
+      />
+    </div>
+    <div>
+      {marketInfo.outcomes.map((outcome: any, index: number) => (
+        <span key={index} className="flex flex-row items-center justify-left">
+          <CompetitorPreview
+            teamId={outcome.teamId}
+            teamContract={teamContract}
+          />
+          <StandardButton
+            variant="contained"
+            onClick={() => buy(outcome.index)}
+            disabled={isMarketClosed || !selectedAmount}
+            className="rounded-full mx-2"
+            backgroundColor="bg-moon-green"
+          >
+            Buy {outcome.probability.toString()}%
+          </StandardButton>
+          <StandardButton
+            variant="contained"
+            className="rounded-full mx-2"
+            backgroundColor="bg-moon-orange"
+            onClick={() => sell(outcome.index)}
+            disabled={isMarketClosed || !selectedAmount}
+            className="rounded-full"
+          >
+            Sell
+          </StandardButton>
+          {isMarketClosed && (
+            <StandardButton
+              variant="contained"
+              onClick={redeem}
+              disabled={!marketInfo.payoutDenominator}
+            >
+              Redeem
+            </StandardButton>
+          )}
+          <div className="mx-2">
+            Balance: {outcome.balance.toPrecision(2).toString()}
+          </div>
+        </span>
+      ))}
+    </div>
+  </>
+)
+const OperatorActions: React.FC<OperatorActionsProps> = ({
+  isMarketClosed,
+  close,
+}) => (
+  <>
+    <h3>Operator actions:</h3>
+    <StandardButton
+      variant="contained"
+      onClick={close}
+      disabled={isMarketClosed}
+    >
+      Close
+    </StandardButton>
+  </>
+)
+
+const OracleActions: React.FC<OracleActionsProps> = ({
+  isMarketClosed,
+  marketInfo,
+  resolve,
+}) => (
+  <>
+    <h3>Oracle actions:</h3>
+    <div>
+      {marketInfo.outcomes.map((outcome: any, index: number) => (
+        <StandardButton
+          key={index}
+          variant="contained"
+          onClick={() => resolve(index)}
+          disabled={!isMarketClosed}
+        >
+          Resolve {outcome.title}
+        </StandardButton>
+      ))}
+    </div>
+  </>
+)
+
+const Layout: React.FC<LayoutProps> = ({
+  account,
+  isConditionLoaded,
+  isMarketClosed,
+  marketInfo,
+  teamContract,
+  setSelectedAmount,
+  selectedAmount,
+  setSelectedOutcomeToken,
+  selectedOutcomeToken,
+  buy,
+  sell,
+  redeem,
+  close,
+  resolve,
+}) => {
+  return (
+    <div>
+      {isConditionLoaded ? (
+        <>
+          <p>Who will be the first team to land on the moon?</p>
```

Comment:
Should we make this a variable / something we can modify for different deprizes? I guess this is fine for the demo for now though

Reply:
yep fixed.